app/views/alchemy/welcome.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Welcome to AlchemyCMS</title>
+    <link href='https://fonts.googleapis.com/css?family=Bitter' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <%= stylesheet_link_tag "alchemy/welcome", media: "all" %>
+  </head>
+  <body>
+    <div class="container">
+      <%= image_tag 'alchemy/alchemy-logo.svg' %>
+      <h3>You have succesfully installed AlchemyCMS!</h3>
+      <p>
+        Congratulations and welcome aboard. First we want to thank you for using Alchemy.
+        We had fun building it and hope you'll have fun working with it.
+      </p>
```

Comment:
"Congratulations! Thank you for using Alchemy. We hope you enjoy using it as much as we do!"